esmvalcore/preprocessor/_regrid.py

```diff
@@ -478,13 +479,12 @@ def regrid(cube, target_grid, scheme, lat_offset=True, lon_offset=True):
         Alternatively, a dictionary with a regional target grid may
         be specified (see above).
 
-    scheme : str
-        The regridding scheme to perform, choose from
-        ``linear``,
-        ``linear_extrapolate``,
-        ``nearest``,
-        ``area_weighted``,
+    scheme : str or dict
+        The regridding scheme to perform. If both source and target grid are
+        structured (regular or irregular), can be one of the built-in schemes
+        ``linear``, ``linear_extrapolate``, ``nearest``, ``area_weighted``,
         ``unstructured_nearest``.
+        Alternatively, a `dict` that specifies generic regridding (see below).
```

Comment:
```suggestion
        Alternatively, a ``dict`` that specifies generic regridding (see below).
```

Reply:
Numpy style docstrings like we use should use single backticks here, I believe (see https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts).